app/tests/e2e/test_issuer.py

```diff
@@ -91,7 +100,11 @@ async def test_get_records(alice_member_client: AsyncClient):
 async def test_send_credential_request(
     bob_member_client: AsyncClient, credential_exchange_id: str
 ):
-    time.sleep(10)
+    assert check_webhook_state(
```

Comment:
I think we can fix these tests now right? With auto-accept disabled and by listening to webhook events?

Reply:
yes we can - in fact I left some out here because I wanted feedback first. Wanted to avoid rewriting it all to potentially re-write it again then in case feedback was more substantial